### PR TITLE
📖 Document how to connect to InMemoryClusters

### PR DESCRIPTION
**Pull request by fabriziopandini**
**What this PR does / why we need it**:

/area documentation

### CODE REVIEW

**Review by chrischdi — COMMENTED** [review]
/lgtm

File: test/infrastructure/inmemory/README.md [hunk 1]
```diff
@@ -90,6 +90,40 @@ CAPIM.clusterclass `quick-start` and CAPIM.template `development` or use your ow
...
+# NOTE: use namespace and name of the cluster you want to connect to
+export NAMESPACE=default 
+export CLUSTER_NAME=in-memory-development-11461
+
+export CONTROL_PLANE_ENDPOINT_PORT=$(k get cluster -n $NAMESPACE $CLUSTER_NAME  -o=jsonpath='{.spec.controlPlaneEndpoint.port}')
+
+kubectl port-forward -n capim-system deployments/capim-controller-manager $CONTROL_PLANE_ENDPOINT_PORT
+```
+
+From terminal window 2:
```
Comment (chrischdi) [inline test/infrastructure/inmemory/README.md:113]: NIT

```suggestion
From terminal window #2:
```

File: test/infrastructure/inmemory/README.md [hunk 2]
```diff
@@ -90,6 +90,40 @@ CAPIM.clusterclass `quick-start` and CAPIM.template `development` or use your ow
...
+
+```shell
+# NOTE: use namespace and name of the cluster you want to connect to
+export NAMESPACE=default 
+export CLUSTER_NAME=in-memory-development-11461
+
+export CONTROL_PLANE_ENDPOINT_PORT=$(k get cluster -n $NAMESPACE $CLUSTER_NAME  -o=jsonpath='{.spec.controlPlaneEndpoint.port}')
+export CONTROL_PLANE_ENDPOINT_IP=$(k get cluster -n $NAMESPACE $CLUSTER_NAME  -o=jsonpath='{.spec.controlPlaneEndpoint.host}')
```
Comment (chrischdi) [inline test/infrastructure/inmemory/README.md:121]: Unused
```suggestion
```

**Comment by sbueringer** [comment]
Thx!

/lgtm
/approve

File: test/infrastructure/inmemory/README.md (hunk 1 above)
Comment (sbueringer) [reply test/infrastructure/inmemory/README.md]: @chrischdi Missed this in CAPV :D

**Pull request merged**